content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+
+Pufferfish is a [fork](#backend-servers) of [Paper](#paper) that adds significant optimizations to the game without much impact to normal gameplay.\
+These optimizations include entity, hopper, SIMD and other performance improvements.\
+This is highly recommended for all servers due to the significant performance improvements it brings, without affecting plugin compatibility.
+
+<Card href="https://pufferfish.host/downloads" title="Download">
+  Download a copy of the latest Pufferfish server software via pufferfish.host.\
+  **Do not attempt to use Pufferfish+ or Pufferfish+ (Purpur). This is a paid
+  server software and is mostly used only by the customers of Pufferfish Host
+  itself**.
+</Card>
+
+## Purpur
+
+[Purpur](https://purpurmc.org/) is another fork of Paper that adds additional configuration concerning different mechanics of the game, removing the need to download or create additional plugins.\
+It also features some performance patches from [Pufferfish](#pufferfish-recommended).
+It is recommended to use Pufferfish over Purpur, unless you require the additional features and configuration that Purpur provides.
+
+<Card
+  href="https://jar.setup.md/download/purpur/latest/latest"
+  title="Download">
+  Download a copy of the latest Purpur server software via
+  [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## Folia
+
+[Folia](https://papermc.io/software/folia) is a fork of Paper that splits the worlds up into "regions" which all run relatively independently.\
+Each region features its own tick loop (ticking at Minecraft's regular 20TPS), running on a thread pool in parallel.\
+This allows for better performance on servers with many players and entities, who are spread far out.
+
+Folia was created because Minecraft is single-threaded, meaning it cannot take advantage of all of your CPU, and instead can only use a small portion of it.\
+Softwares such as Paper already help to alleviate this issue, but Folia takes it a step further by splitting the world up into regions.
+
+<Callout type="warn" title="Warning">
+    **Folia is not completely usable!**\
+    You must consider whether you need to use it. Folia is not completely stable yet, although it is stable enough to be used in production, one famous example being 2b2t.\
+    You must be prepared to deal with bugs and issues and a high hardware requirement of a (recommended) minimum of 16 CPU cores.
+    Consider using other (more stable) softwares such as [Pufferfish](#pufferfish-recommended) or [Purpur](#purpur) beforehand.
+
+    **Ensure you thoroughly test every feature, command and mechanic in your server while using this software.**
+
+    Additionally, **Folia does not fix lag**, and requires a very specific playerbase where everyone is spread out relatively far.
+    If your players are not spread out then there will be more entities in each region, which will cause more lag, making the experience identical if not worse than using a software such as Paper.
```

Comment:
"A software" is not proper English grammar. "Software" is an uncountable noun, like "water" or "air". Drop the "a".

Reply:
Resolved in [315fbf0](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/315fbf0ca07c1376b029650cff9edfc7152a2267)